Cargo.toml

```diff
@@ -3,6 +3,10 @@ name = "versionize_derive"
 version = "0.1.1"
 license = "Apache-2.0"
 authors = ["Amazon Firecracker team <firecracker-devel@amazon.com>"]
+description = "Implements the Versionize derive proc macro."
+readme = "README.md"
+repository = "https://github.com/firecracker-microvm/versionize_derive"
+keywords = ["serialization", "version"]
 
```

Comment:
nit: maybe add `categories = ["encoding", "parsing"]` ?

Reply:
These are proper keywords. I think we need to dive a little deeper into this SEO thing, and optimize later.